src/parser.js

```diff
@@ -154,6 +154,11 @@ export default class Parser {
 
   // == Internal properties & functions ==
 
+  // Base path (URL) name for the API
+  get basePath() {
+    return (this.swagger.basePath || '').replace(/[/]+$/, '');
```

Comment:
The above behavior is correct according to the spec as far as I can tell:

> The field name MUST begin with a slash. The path is appended to the basePath in order to construct the full URL.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#paths-object